docs/articles/documentation/guides/basic-guides/assert.md

```diff
@@ -138,6 +133,45 @@ To set the timeout when you launch tests, pass the timeout value to the [runner.
 method if you use API or specify the [assertion-timeout](../../reference/command-line-interface.md#--assertion-timeout-ms) option
 if you run TestCafe from the command line.
 
+### Smart Assertion Limitations
+
+Smart assertions auto-retry only works with:
```

Comment:
Can you clarify what you mean by "Smart assertions auto-retry"?

Reply:
It's a feature. Since we don't really use the word, I'd say:

> Smart assertions auto-retry mechanism only works with:

Reply:
The smart assertion's auto-retry mechanism only works with:

Reply:
Edited